### ISSUE TITLE
iv_robust()

### ISSUE THREAD

**Issue by medinali** [DeclareDesign/estimatr#189]
Hi -- 

I'm trying to use iv_robust to estimate the regression below

`iv_robust(uptake ~ heard + bin | uspeak_any + bin, data = dat, weights = IPW_any, se_type = "stata")
`
dat = [](url) [MyData_dd.xlsx](https://github.com/DeclareDesign/estimatr/files/1852021/MyData_dd.xlsx)

Unfortunately, I get the following error message

> Error in solve.default(vcov_fit$Vcov_hat[indices, indices], fit$beta_hat[indices]) : 
>   system is computationally singular: reciprocal condition number = 2.42888e-18

I was wondering if there is a way to fix this or if possible to report a more informative message.
 I'm guessing the problem is coming from the computation of the F-statistic; also noticed that iv_robust *seems* to be computing the vcov_hat. 

Thanks

**Comment by lukesonnet** [comment]
Hi, thank you for the bug report!

**Solution**: I'm not sure that we can solve this problem at the moment in order to compute an f-statistic, although we are investigating. At the very least, we will warn instead of erroring during the f-statistic computation so that you can still get estimates and standard errors where possible. This should be coming very soon.

The cause of the problem seems to be the fact that some of your `bin`s are of size one, leading to a variance covariance matrix that is rank-deficient. I note that the error also occurs when we use `car` to get the F-statistic (`car` is what `AER::ivreg` uses) if we pass it the "HC1" (or "stata") variance covariance matrix.

**Comment by lukesonnet** [comment]
For now you can install from a branch that skips computing the f-stat in this case by doing the following:

```r
devtools::install_github("DeclareDesign/estimatr@ls/iv_fstat")
```

**Comment by medinali** [comment]
This is great! Thank you.

**Comment by nfultz** [comment]
Fixed in v0.7, which should be propagating to the DD drat imminently. Thanks much @lukesonnet .
